src/MovieList.jsx

```diff
@@ -132,15 +132,67 @@ function MovieList({
           Search🔍
         </button>
       </div>
-      <div
-        className={
-          showSearch ? "movieListContainerInactive" : "movieListContainerActive"
-        }
-      >
-        {divs}
-        <button onClick={() => setPage(page + 1)} className="loadMoreButton">
-          Load More
-        </button>
+      <div className="reccomendations">
+        <div className="recommendationsBanner">
+          <h1>Personalized Recommendations</h1>
+          <button onClick={() => navigate("../recommendations")}>
+            Show all
+          </button>
+        </div>
+        <div
+          className={
+            showSearch
+              ? "movieListContainerInactive"
+              : "movieListContainerActive"
+          }
+        >
+          {divs}
+        </div>
+      </div>
+      <div className="My dynamic rankings">
+        <div className="recommendationsBanner">
+          <h1>My Rankings</h1>
+          <button onClick={() => navigate("../myRankings")}>Show all</button>
+        </div>
+        <div
+          className={
+            showSearch
+              ? "movieListContainerInactive"
+              : "movieListContainerActive"
+          }
+        >
+          {divs}
+        </div>
+      </div>
+      <div className="bookmarks">
+        <div className="recommendationsBanner">
+          <h1>Bookmarked Movies</h1>
+          <button onClick={() => navigate("../Bookmarks")}>Show all</button>
+        </div>
+        <div
+          className={
+            showSearch
+              ? "movieListContainerInactive"
+              : "movieListContainerActive"
+          }
+        >
+          {divs}
```

Comment:
The exact same content is in three places. Is this going to change? How?

Reply:
When you hit the search bar everything goes inactive. Does that answer your question